### PR TITLE
Feature/cka implement new fintech ui

### PR DESCRIPTION
**Pull request by collins-k**

### CODE REVIEW

File: fintech-examples/fintech-ui/src/app/bank-search/common/search/search.component.scss [hunk 1]
```diff
@@ -17,6 +19,6 @@
     padding-top: 18px;
```
Comment (vladimir-nechai) [inline fintech-examples/fintech-ui/src/app/bank-search/common/search/search.component.scss]: why not to use rem or em instead of pixels ?

Comment (collins-k) [reply fintech-examples/fintech-ui/src/app/bank-search/common/search/search.component.scss]: for the search I want the unit to be fix.  We don't need a relative unit there

File: fintech-examples/fintech-ui/src/app/bank-search/components/bank-search/bank-search.component.html [hunk 2]
```diff
@@ -1,23 +1,12 @@
...
-  <div class="box-item mx-4">
-    <h4 class="box-header__title">Please search your bank</h4>
-    <p class="box-header_text">
-      Lorem ipsum dolor sit amet, consetetur sadipscing elitr sed diam nonumy eirmod tempor invidunt
-    </p>
-  </div>
-
   <ng-template [ngIf]="!selectedBank" [ngIfElse]="profile">
-    <div class="card box-item mr-4 w-100">
+    <div class="card card-frame">
```
Comment (vladimir-nechai) [inline fintech-examples/fintech-ui/src/app/bank-search/components/bank-search/bank-search.component.html]: is it a bootstrap class?

File: fintech-examples/fintech-ui/src/app/login/login.component.html [hunk 3]
```diff
@@ -1,13 +1,13 @@
 <form (ngSubmit)="onSubmit()" [formGroup]="loginForm">
-  <div class="card">
+  <div class="card card-frame">
```
Comment (vladimir-nechai) [inline fintech-examples/fintech-ui/src/app/login/login.component.html]: Is it a bootstrap class?

Comment (collins-k) [reply fintech-examples/fintech-ui/src/app/login/login.component.html]: Nope! A custom class I created for login and bank-search card. Take a look at style.scss when you want to know what this does

**Pull request merged**